### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RosaTheDev**
What (if any) features are you implementing?
- Updating the README, Made it more specific in order to meet the requirements in the spec
What (if anything) did you refactor?
- Improved upon the README and added images + gifs for user choice
Were there any issues that arose?
- N/A
Is there anything that you need from your teammate?
- Final look though & approval. 
- I would like to add images in the README i.e LoFi coding gifs just to make it more fun to read 
Any other comments, questions, or concerns?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,60 @@
...
+ * Every time the user clicks the Show New Random Cover button or refreshes the page, a new random cover is created & displayed
+ * When a user clicks the “Make Your Own Cover” button, we should see the form, and the user can create their own book cover
+ * When the Make My Book button is clicked, the site should show the new cover that is created
+ * When a user clicks the “Save Cover” button, the current cover will be added and stored in the "View Saved Covers" page
+ * When a user clicks the “View Saved Covers” button, we should see all the saved covers 
+ * From the saved covers view, if a user double clicks a saved cover, it will be deleted
+
+## How To Run & View 📽
+* To Run our site click on the Deploy Link below 👇🏾
+* https://gabby-recny.github.io/romcom/
```
Comment (gjtrecny) [inline README.md]: Try to convert the phrase "deploy link" to the link.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,60 @@
...
+
+
+## Want Some Images To Use As Covers? 📚
+* When you go to make your own cover the first text form that says **Cover** needs a link/URL for the image! We provided some that are pretty diverse! 👇🏾
+
+* Lofi gif:
```
Comment (gjtrecny) [inline README.md]: Same goes for these links

File: README.md [hunk 3]
```diff
@@ -1,25 +1,60 @@
...
+* HTML
+* CSS
+* JavaScript
+
+## Learning Goals
  * Write clean, DRY JavaScript to store our data
  * Use a provided class by creating object instances using the new keyword
  * Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
 
+## What Should Happen? 👨🏾‍💻 👩🏾‍💻
```
Comment (gjtrecny) [inline README.md:17]: Go into more detail on functionality

**Review by Gabby-Recny — CHANGES_REQUESTED** [review]

**Comment by Gabby-Recny** [comment]
Thanks for resolving those issues! Good work!

**Pull request merged**